packages/round-manager/src/features/round/ViewRoundSettings.tsx

```diff
@@ -0,0 +1,519 @@
+              defaultValue={`${getUTCDate(round.roundEndTime)} ${getUTCTime(
+                round.roundEndTime
+              )}`}
+              disabled
+            />
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+function Funding(props: { round: Round }) {
+  const { round } = props;
+
+  const matchingFundPayoutToken =
+    props.round &&
+    payoutTokens.filter(
+      (t) => t.address.toLocaleLowerCase() == round.token.toLocaleLowerCase()
+    )[0];
+
+  const matchingFunds =
+    (props.round &&
+      props.round.roundMetadata.quadraticFundingConfig
+        ?.matchingFundsAvailable) ??
+    0;
+
+  return (
+    <div className="w-full">
+      <span className="mt-4 inline-flex text-lg font-light text-gray-600 mb-4">
+        Funding Amount
+      </span>
+      <div className="grid grid-cols-2 grid-rows-1 gap-4 mb-4">
+        <div>
+          <div
+            className={
+              "text-sm leading-5 font-semibold pb-1 flex items-center gap-1 mb-2"
+            }
+          >
+            Payout Token
+          </div>
+          <div className={"leading-8 font-normal text-grey-400"}>
+            <input
+              type="text"
+              className="w-full rounded-md border border-gray-300 shadow-sm py-2 px-3 bg-white text-sm leading-5 focus:outline-none focus:ring-blue-500 focus:border-blue-500 transition duration-150 ease-in-out"
+              defaultValue={matchingFundPayoutToken.name}
+              disabled
+            />
+          </div>
+        </div>
+        <div>
+          <div
+            className={
+              "text-sm leading-5 font-semibold pb-1 flex items-center gap-1 mb-2"
+            }
+          >
+            Matching Funds Available
+          </div>
+          <div className={"leading-8 font-normal text-grey-400"}>
+            <input
+              type="text"
+              className="w-2/12 rounded-l-md border border-gray-300 shadow-sm py-2 px-3 bg-white text-sm leading-5 focus:outline-none focus:ring-blue-500 focus:border-blue-500 transition duration-150 ease-in-out"
+              defaultValue={matchingFundPayoutToken.name}
+              disabled
+            />
+            <input
+              type="text"
+              className="w-10/12 rounded-r-md border border-gray-300 shadow-sm py-2 px-3 bg-white text-sm leading-5 focus:outline-none focus:ring-blue-500 focus:border-blue-500 transition duration-150 ease-in-out"
+              defaultValue={matchingFunds}
+              disabled
+            />
+          </div>
+        </div>
+      </div>
+
+      <span className="mt-4 inline-flex text-lg font-light text-gray-600 mb-4">
+        Matching Cap
+      </span>
+      <div className="grid grid-cols-2 grid-rows-1 gap-4 mb-4">
+        <div>
+          <div
+            className={
+              "text-sm leading-5 font-semibold pb-1 flex items-center gap-1 mb-2"
+            }
+          >
+            Do you want a matching cap for projects?
```

Comment:
If there is no matching cap, the page is failing to load with the following error:

ViewRoundSettings.tsx:366 Uncaught TypeError: Cannot read properties of undefined (reading 'matchingCap')